### PR TITLE
Check if TTY exist before we get screen size - Second PR try

### PR DESCRIPTION
**Pull request by bolausson**
**By submitting this pull request, I confirm the following:** 
*please fill any appropriate checkboxes, e.g: [X]*

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [X] I have made only one major change in my proposed changes.
- [X] I have commented my proposed changes within the code.
- [X] I have tested my proposed changes, and have included unit tests where possible.
- [X] I am willing to help maintain this change if there are issues with it later.
- [X] I give this submission freely and claim no ownership.
- [X] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [X] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

Please make sure you [Sign Off](https://github.com/pi-hole/pi-hole/wiki/How-to-signoff-your-commits.) all commits. Pi-hole enforces the [DCO](https://github.com/pi-hole/pi-hole/wiki/Contributing-to-the-project).

---
**What does this PR aim to accomplish?:**
This change fixes issue #145 "stty: standard input: Inappropriate ioctl for device "

**How does this PR accomplish the above?:**
It checks if a real terminal exist, if not it sets the screen size to a fixed value.

**What documentation changes (if any) are needed to support this PR?:**
No changes to the documentation are required

---
* You must follow the template instructions. Failure to do so will result in your pull request being closed.
* Please respect that Pi-hole is developed by volunteers, who can only reply in their spare time.

### CODE REVIEW

**Comment by PromoFaux** [comment]
@bolausson had to do some trickery to sort it out. You were very close, but you were still basing you work on the `master` branch.

Rather than get you to do it all over again, I did the following:

```
git clone git@github.com:bolausson/pi-hole.git
cd pi-hole
git checkout development
git checkout -b mybranch #this creates a new local branch on my workstation called mybranch
git cherry-pick aedb37b0d396bc27555d289e042a630fe85398dd #This adds a copy of your commit onto this fresh new version of mybranch
git push origin mybranch --force #This was a bit naughty, but it essentially overwrites mybranch on the remote with the version that I just created.
```

The issue before, where you were basing on `master`, is that there are still merge commits and tags in `master` that are not to go into `development`. No biggy, but figured it was probably easier to do the hard part for you, and then explain it, rather than have to tell you to do it all over again ;)

Any questions, please ask!

**Comment by bolausson** [comment]
@PromoFaux thanks a lot!

I still don't know why it was still based on the master *sigh*
I did run the following after forking pi-hole via the website:
```
git clone https://github.com/bolausson/pi-hole.git
git checkout -b development
git checkout -b mybranch development
git add basic-install.sh
git commit --signoff --message '...'
```
After that I created a pull request and made sure this time I select pi-hole:development.

Can you tell me where it did went wrong?

Cheers,
Bjoern

**Comment by PromoFaux** [comment]
> ```
> git checkout -b development
> git checkout -b mybranch development
> ```

Honest mistake to make! So in plain english, what you told `git` to do here, keeping in mind `master` is the default branch:

```
- Make a new local branch called `development` based on the branch I'm currently on (`master`)
- Make a new local branch called  `mybranch` based on my local branch `development` (which is infact `master`, but with the name `development`!)
```

Basically, you could either have run:

```
git checkout development #note lack of `-b`
git checkout -b mybranch
```

or

```
git checkout -b mybranch development
```

Out of the two ways I've presented, I prefer to go with the first, as its more verbose, and not that much extra typing when you can just tab-out the commands as you go!

**Comment by bolausson** [comment]
I see.

Thanks a lot for the detailed explanation!

Cheers,
Bjoern
